Detections/MultipleDataSources/TimeSeriesAnomaly-MultiVendor_DataExfiltration.txt

```diff
@@ -0,0 +1,40 @@
+// Name: Time Series Anomaly detection for size of data transferred to Public Internet
```

Comment:
Looks good.

Reply:
Meant to indicate that the query is fine, but lets add some context in such as SourceIP, DestIP, Port and maybe Count of them so it is understood what makes up the spike traffic, this will save the customer time in investigating.

Reply:
Added query secctions to populate more context in results to assist investigations.

Reply:
cool